app/index.jsx

```diff
@@ -17,10 +32,11 @@ let App = React.createClass({
     this.setState({ selectedElementIndex: clickedLiIndex })
   },
   render: function () {
-    let featureTitle = constants.elements[this.state.selectedElementIndex]
+    const featureTitle = constants.elements[this.state.selectedElementIndex]
+    const Component = componentCode[this.state.selectedElementIndex]
```

Comment:
rename to `const BootstrapElement` or leave it as `BootstrapPre`?
Seems like a better name than just `Component`.